src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -347,53 +348,86 @@ public void Dispose()
                 Assert.Equal(new byte[bufferSize], buffer);
 
                 context.Response.Close();
+                await clientTask;
             }
         }
 
         [Fact]
         public async Task CanSeek_Get_ReturnsFalse()
         {
-            HttpListenerRequest response = await _helper.GetRequest(chunked: true);
```

Comment:
@caesar1995 I don't see other references to _helper yet it's still defined - any reasons?